core/node/state_keeper/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub use self::{
 };
 
 pub mod executor;
+pub mod health;
```

Comment:
Nit: Does this module need to be public?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3193/commits/9fe98d2fd980f07bc741178d93fb07d78099b7ee